<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>My_Qt_PlayerClass</class>
 <widget class="QMainWindow" name="My_Qt_PlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My_Qt_Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="OpenFile_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play_Button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause_Button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop_Button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume_slider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volume_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;volume&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="audio_list_table_widget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>175</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>41</number>
    </attribute>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="Add_new_audio_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="call_yt_dlp_debug_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>call yt-dlp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="url_line_edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>11</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="url_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;url&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>Play_Button</zorder>
   <zorder>Pause_Button</zorder>
   <zorder>OpenFile_button</zorder>
   <zorder>Stop_Button</zorder>
   <zorder>volume_slider</zorder>
   <zorder>volume_label</zorder>
   <zorder>audio_list_table_widget</zorder>
   <zorder>Add_new_audio_button</zorder>
   <zorder>call_yt_dlp_debug_button</zorder>
   <zorder>url_line_edit</zorder>
   <zorder>url_label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="My_Qt_Player.qrc"/>
 </resources>
 <connections/>
</ui>
